pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -112,6 +112,8 @@ type VirtualCenterConfig struct {
 	DatacenterPaths []string
 	// TargetDatastoreUrlsForFile represents URLs of file service enabled vSAN datastores in the virtual center.
 	TargetvSANFileShareDatastoreURLs []string
+	// TargetvSANFileShareClusters represents file service enabled vSAN clusters on which file volumes can be created.
+	TargetvSANFileShareClusters []string
```

Comment:
`TargetvSANFileShareDatastoreURLs` already contains a list of vSAN datastore URLs on which file service is enabled. 
Why do we need one more field `TargetvSANFileShareClusters`? Having two fields for the same purpose will create confusion.

Reply:
So this new field will be set when file volume services are enabled by user in WCP. There's an API that accepts the list of cluster IDs. Now `wcpsvc` doesn't want the logic to fetch datastores corresponding to these clusters in their layer. They just agreed to pass the user input to CSI without any additional handling.
That's the reason we're introducing this new field. I have also updated the PR description for this.

Reply:
So `TargetvSANFileShareClusters` should not be used in the vanilla setup? Can we add validation to disallow using this config for vanilla deployment?

Reply:
Yes it should not be used in vanilla setup. And we will not be adding info about this field in our fields.
Even if it's mentioned for vanilla, there will be no effect as we're not using it in vanilla. Should we still add the validation?